### DISCUSSION TITLE
Import i18n component ?

### DISCUSSION THREAD

**Discussion by Ericlm** [fluent-vue/fluent-vue#812] [Q&A] [answered]
Hello! I'm working on a Vue 3 with Composition API project. I'm currently trying to use the `<i18n>` component, and I would like to get rid of an eslint warning (`eslint(vue/no-undef-components)`). I guess I could remove the warning by tweaking eslint config, but I was wondering if there is a way to actually import this component ? And beneficiate of props auto-completion at the same time ? 😀

**Comment by Demivan** [comment, marked as answer]
Hi

`<i18n>` component is automatically imported and registered globally when `app.use(fluent)` is called. eslint is not aware of that, so the only way to fix eslint is to add this component as an exception to the `vue/no-undef-components` rule.

Props autocompletion and typechecking is an separate issue. Please create an issue in https://github.com/fluent-vue/fluent-vue repository.

**Reply by Ericlm** [reply]
I'll go with the ignore eslint warning, thank you for your time!
